src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const longTermDiscount = 50;
+  const normalTermDiscount = 20;
+  const rentalCost = 40;
+  const longTerm = 7;
+  const normalTerm = 3;
+  let totalCost = days * rentalCost;
+
+  if (days >= longTerm) {
+    totalCost -= longTermDiscount;
+  }
+
+  if (days >= normalTerm && days < longTerm) {
+    totalCost -= normalTermDiscount;
```

Comment:
Since you are returning from the function after applying the long term discount, you don't need to check if days is less than `longTerm` in the second `if` statement. The code will not reach this point if the first condition is true, so you can simplify the condition to just check if `days` is greater than or equal to `normalTerm`.